packages/esm-esbuild/esbuild.js

```diff
@@ -0,0 +1,21 @@
+import { build } from "esbuild";
+
+build({
+  entryPoints: ["./src/index.ts"],
+  outdir: "./dist",
+  bundle: true,
+  // platform: "node",
+  format: "esm",
+  // target: "node16",
+  plugins: [
+    {
+      name: "add-mjs",
```

Comment:
I actually tried to make it more sophisticated by adding `.mjs` only for mappersmith imports:
```javascript
plugins: [
    {
      // Temporary patch for the recent mappersmith release, based on this response: https://github.com/evanw/esbuild/issues/622#issuecomment-769462611
      name: 'add-mjs-to-mappersmith',
      setup(build) {
        build.onResolve({ filter: /.*/ }, (args) => {
          if (
            args.resolveDir.endsWith('/node_modules/mappersmith/esm') &&
            !args.path.endsWith('.mjs')
          ) {
            return { path: args.path + '.mjs', external: true }
          }
        })
      },
    },
  ],
```